SPEC.md

```diff
@@ -1352,13 +1352,55 @@ Loops are distinct from scatter clauses because the body of a while loop needs t
 
 ### Conditionals
 
-:pig2: Coming soon in [Cromwell](https://github.com/broadinstitute/cromwell)
+:pig2: Available in [Cromwell](https://github.com/broadinstitute/cromwell) version 24 and higher
 
 ```
 $conditional = 'if' '(' $expression ')' '{' $workflow_element* '}'
 ```
 
-Conditionals only execute the body if the expression evaluates to true
+Conditionals only execute the body if the expression evaluates to true.
+
+* When a call's output is referenced outside the same containing `if` it will need to be handled as an optional type. E.g.
+```
+workflow foo {
+  # Call 'x', producing a Boolean output:
+  call x
+  Boolean x_out = x.out
+
+  # Call 'y', producing an Int output, in a conditional block:
+  if (x_out) {
+    call y
+    Int y_out = y.out
+  }
+  
+  # Outside the if block, we have to handle this output as optional:
+  Int? y_out_maybe = y.out
+
+  # Call 'z' which takes an optional Int input:
+  call z { input: optional_int = y_out_maybe }
```

Comment:
Correct! I only called it out in the separate line to show that the type has changed